src/components/Grid/Grid.tsx

```diff
@@ -121,6 +121,9 @@ export const Grid: React.FC<GridProps> = ({
             item={item}
             grid={grid}
             onClick={() => onClick(item)}
+            onKeyUp={event => {
+              if (event.key === "Enter") onClick(item);
```

Comment:
Buttons should also be triggered by space.

```suggestion
              if (event.key === "Enter" || event.code === "Space") onClick(item);
```